### PR TITLE
add Nikita's hackbar

### PR DESCRIPTION
**Pull request by Nikita-kumari**
Create hacktoberfest2020 file

### CODE REVIEW

**Comment by Nikita-kumari** [comment]
please pull my request...

File: _hackbars/Nikita-kumari.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+layouts: hackbar
+title: hacktoberfest
+author: Nikita kumari
+
+"Hacktoberfest is a promotion that Digital Ocean runs to encourage developers to contribute patches to open-source..."
+
+---
+
+## <!-- Climate change will impact the lives of future generations...
+![climate change canvas]({{https://www.foundmyself.com/images/blank.png}}https://www.foundmyself.com/images/Nikita_kumari.png
```
Comment (tasnimzotder) [inline _hackbars/Nikita-kumari.md]: hey @Nikita-kumari, you need to add an image in the `assets/images` directory and link the image here.

For more details you can go through the document.

**Comment by tasnimzotder** [comment]
Yes. But you need to add the image `Nikita_kumari.jpg` to the directory `assets/images/`.

**Comment by Nikita-kumari** [comment]
Done everything ,please accept my request

File: _hackbars/Nikita-kumari.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+layouts: hackbar
+title: hacktoberfest
+author: Nikita kumari
+
+"Hacktoberfest is a promotion that Digital Ocean runs to encourage developers to contribute patches to open-source..."
+
+---
+
+## <!-- Climate change will impact the lives of future generations...
+![climate change canvas]({{site.baseurl}}}/assets/images/Nikita_kumari.jpg)
```
Comment (tasnimzotder) [inline _hackbars/Nikita-kumari.md]:
```suggestion
![climate change canvas]({{site.baseurl}}}/assets/images/Nikita-kumari.jpg)
```

**Review by tasnimzotder — APPROVED** [review]
Everything is okay 👌

**Comment by Nikita-kumari** [comment]
Thank you....

**Comment by tasnimzotder** [comment]
Welcome and thanks for your contributions :)

**Comment by tasnimzotder** [comment]
Oops..... You missed another line.

**Comment by Nikita-kumari** [comment]
Which line 😑?

**Comment by tasnimzotder** [comment]
@Nikita-kumari, you can correct the following and open another PR. Otherwise, your hackbar won't be live.
The error is in line 5 and 10.

Current version:
```markdown
---
layouts: hackbar
title: hacktoberfest
author: Nikita kumari

"Hacktoberfest is a promotion that Digital Ocean runs to encourage developers to contribute patches to open-source..."

---

## <!-- Climate change will impact the lives of future generations...
![climate change canvas]({{site.baseurl}}}/assets/images/Nikita-kumari.jpg)
```

Correct version:
```markdown
---
layouts: hackbar
title: hacktoberfest
author: Nikita kumari
---

"Hacktoberfest is a promotion that Digital Ocean runs to encourage developers to contribute patches to open-source..."

---

## Climate change will impact the lives of future generations...
![climate change canvas]({{site.baseurl}}}/assets/images/Nikita-kumari.jpg)
```

**Comment by Nikita-kumari** [comment]
Everything once  again?

**Comment by tasnimzotder** [comment]
No. Just do the following steps in your local machine (laptop) -

1. Open the file `_hackbars/Nikita-kumari.md`
2. Remove all the codes in that file.
3. Paste the following codes -
```markdown
---
layouts: hackbar
title: hacktoberfest
author: Nikita kumari
---

"Hacktoberfest is a promotion that Digital Ocean runs to encourage developers to contribute patches to open-source..."

---

## Climate change will impact the lives of future generations...
![climate change canvas]({{site.baseurl}}}/assets/images/Nikita-kumari.jpg)
```
4. Commit the changes
5. Open a pull request
6. 🎉🎉 Done

* Let me know if you need more help in it.

**Pull request merged**